vaccine_feed_ingest/vial.py

```diff
@@ -169,16 +187,23 @@ def search_source_locations(
 
     query = urllib.parse.urlencode(params)
 
-    resp = vial_http.request(
-        "GET", f"/api/searchSourceLocations?{query}", preload_content=False
-    )
+    path_and_query = f"/api/searchLocations?{query}"
```

Comment:
This is wrong! We need to grab existing matches from `source_locations` table, not existing locations

Reply:
*great* catch, thank you.